src/Facility.ConformanceApi/Answer.g.cs

```diff
@@ -17,25 +17,43 @@ namespace Facility.ConformanceApi
 	[JsonConverter(typeof(AnswerJsonConverter))]
 	public partial struct Answer : IEquatable<Answer>
 	{
+		private static readonly Dictionary<string, string> s_normalizedConstants = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
```

Comment:
This should be moved below `s_values` and be initialized similarly (not with a static constructor).

Reply:
FWIW, this was a remnant of an attempt to deduce C#'s initialization order semantics without documentation. It's gone now.